binding.js

```diff
@@ -17,6 +17,13 @@ var minimumLibvipsVersion = process.env.npm_package_config_libvips || require('.
 
 var vipsHeaderPath = path.join(__dirname, 'include', 'vips', 'vips.h');
 
+var platform = process.env.npm_config_platform || process.platform;
+
+var arch = process.env.npm_config_arch || process.arch;
+if (arch === 'armhf') arch = 'arm';
```

Comment:
I'll add that later today, can also probably set version to 8 if value is `aarch64`, I guess there is `armel` but think that's considered obsolete now.